train:
  batch_size: 64
  epochs: 150
  learning_rate: 0.0002
  shuffle: true
  model_filename: "simple_imitation_model.pt"
  shift_signs: [] # アフィン変換
  vel_offset: 0.4
  projection_signs: [-1.0, 0.0, 1.0]  # 射影変換
  projection_offset: 0.2
  save_intermediate_models: true
  tensorboard_logging: true